deepinv/datasets/urban100.py

```diff
@@ -0,0 +1,135 @@
+from typing import Any, Callable
+import os
+
+from PIL import Image
+import torch
+
+from .utils import calculate_md5_for_folder, download_zipfile, extract_zipfile
+
+
+class Urban100HR(torch.utils.data.Dataset):
+    """Dataset for `Urban100 <https://paperswithcode.com/dataset/urban100>`_.
+
+    The Urban100 dataset contains 100 images of urban scenes.
+    It commonly used as a test set to evaluate the performance of super-resolution models.
+
+
+    **Raw data file structure :** ::
+
+        self.root --- image_SRF_2 --- img_001_SRF_2_A+.png
+                   |               |
+                   |               -- img_100_SRF_2_SRCNN.png
+                   |
+                   -- image_SRF_4 --- img_001_SRF_4_A+.png
+                   |               |
+                   |               -- img_100_SRF_4_SRCNN.png
+                   -- readme.txt
+                   -- source_selected.xlsx
+                   -- Urban100_SR.zip
+
+    For each image in the dataset, several variations of the same image exist.
+    For this class, we are dealing with only high resolution images in `image_SRF_4` folder.
+    For more information about the different versions, look at `readme.txt`.
+
+    >>> root = "/path/to/dataset/Urban100"
+    ... dataset = Urban100HR(root=root, download=True)  # will download dataset at root
```

Comment:
the ... should be >>> I believe, since they represent a new line.

Reply:
Also please add a separator |sep| and Example as in [here](https://github.com/deepinv/deepinv/blob/01a7c863438117c7ab52f20be780bbec0b6b749e/deepinv/physics/blur.py#L156).

Reply:
also in the example I would print the dataset length and check that is 100